packages/@webex/plugin-meetings/src/media/index.ts

```diff
@@ -160,24 +157,11 @@ Media.createMediaConnection = (
 
   const iceServers = [];
 
-  // we might not have any TURN server if TURN discovery failed or wasn't done or
-  // we might get an empty TURN url if we land on a video mesh node
-  if (turnServerInfo?.url) {
-    if (!isBrowser('firefox')) {
-      let bareTurnServer = turnServerInfo.url;
-      bareTurnServer = bareTurnServer.replace('turns:', 'turn:');
-      bareTurnServer = bareTurnServer.replace('443', '5004');
-
-      iceServers.push({
-        urls: bareTurnServer,
-        username: turnServerInfo.username || '',
-        credential: turnServerInfo.password || '',
-      });
-    }
-
+  // we might not have any TURN server if TURN discovery failed or wasn't done or we land on a video mesh node
+  if (turnServerInfo?.urls.length > 0) {
```

Comment:
previously when we got an empty url (happens on VMN), it was just passed on from turnDiscovery.ts and reached all the way here and checked here with the check `if (turnServerInfo?.url)`. Now I've moved the filtering out of empty urls into handleTurnDiscoveryResponse() in turnDiscovery.ts so here we only need to check the length of urls array